xml/System.Runtime.InteropServices/CollectionsMarshal.xml

```diff
@@ -178,15 +178,15 @@ Items should not be added to or removed from the <xref:System.Collections.Generi
         <Parameter Name="exists" Type="System.Boolean" RefType="out" Index="2" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TKey">To be added.</typeparam>
-        <typeparam name="TValue">To be added.</typeparam>
-        <typeparam name="TAlternateKey">To be added.</typeparam>
-        <param name="dictionary">To be added.</param>
-        <param name="key">To be added.</param>
-        <param name="exists">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TKey">The type of the keys in the dictionary.</typeparam>
+        <typeparam name="TValue">The type of the values in the dictionary.</typeparam>
+        <typeparam name="TAlternateKey">The type of the alternate key in the dictionary lookup.</typeparam>
+        <param name="dictionary">The dictionary to get the ref to <typeparamref name="TValue"/> from.</param>
+        <param name="key">The key used for lookup.</param>
+        <param name="exists">Whether or not a new entry for the given key was added to the dictionary.</param>
+        <summary>Gets a ref to a <typeparamref name="TValue"/> in the <see cref="T:System.Collections.Generic.Dictionary`2.AlternateLookup`1"/>, adding a new entry with a default value if it does not exist in the <paramref name="dictionary"/>.</summary>
```

Comment:
Do you know if `T:System.Collections.Generic.Dictionary`2.AlternateLookup`1` existed in Preview6? Because currently the CI cannot find it. If we introduced it in Preview7 then maybe we will need to merge this after we update dotnet-api-docs to Preview7, to prevent showing something that isn't released yet.

But if it existed in Preview6 already, then I'm not sure what the problem could be. The DocID is well formed, it matches the official DocID: https://github.com/dotnet/dotnet-api-docs/blob/55322a04db5cc185736be7e3cbf56f87cd75ba17/xml/System.Collections.Generic/Dictionary%602%2BAlternateLookup%601.xml#L4

Reply:
I'm not sure. Honestly CollectionsMarshal isn't in my wheelhouse. I had the wrong format beforehand so lets see if CI likes it after I changed it to match the doc id.

Reply:
Alternatively, instead of making these `see crefs` they could just be `<c></c>` for now.

What would be your preference, @gewarren?